core/server/api_container/server/startosis_engine/startosis_constants/startosis_constants.go

```diff
@@ -12,4 +12,7 @@ const (
 	PackageIdPlaceholderForStandaloneScript                          = "DEFAULT_PACKAGE_ID_FOR_SCRIPT"
 	PlaceHolderMainFileForPlaceStandAloneScript                      = ""
 	ParallelismParam                            StarlarkContextParam = "PARALLELISM"
+
+	// DefaultPersistentDirectorySize 500 Megabytes is the default value
+	DefaultPersistentDirectorySize int64 = 500 * 1024 * 1024
```

Comment:
I wonder if this default value could be moved some layers down,  to use the same if the volume is not created from a startosis code, but I don't have a strong conviction I'm just suggesting, probably you already reviewed this, so this is up to you.